CodeEditor/Sources/CodeEditor/LighbulbButton.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  LightbulbButton.swift
+//  
+//
+//  Created by Andreas Cselovszky on 31.01.23.
+//
+
+import UIKit
+import SwiftUI
+
+final class LightbulbButton: UIButton {
+    
+    let setSelectedFeedbackSuggestionId: () -> Void
+    let toggleShowAddFeedback: () -> Void
+    
+    init(frame: CGRect, setSelectedFeedbackSuggestionId: @escaping () -> Void, toggleShowAddFeedback: @escaping () -> Void) {
+        self.setSelectedFeedbackSuggestionId = setSelectedFeedbackSuggestionId
+        self.toggleShowAddFeedback = toggleShowAddFeedback
+        super.init(frame: frame)
+        setup()
+    }
+    
+    required init?(coder: NSCoder) {
```

Comment:
```suggestion
    @available(*, unavailable)
    required init?(coder: NSCoder) {
```

SwiftLint complains without an annotation